src/main/java/org/myrobotlab/service/config/Gpt3Config.java

```diff
@@ -0,0 +1,25 @@
+package org.myrobotlab.service.config;
+
+public class Gpt3Config extends ServiceConfig {
+
+  public String currentBotName = "Alice";
```

Comment:
Why alice?

Reply:
There are probably vars and behaviors that probably should work the same way between ProgramAB and ChatGpt3.  
I guess this is a reminder to start heading in that direction.  Heh, a little experimentation came up with a non-personalized response. 
![image](https://user-images.githubusercontent.com/1266144/213868633-b972c6fd-3757-4c37-bc08-7014995e4110.png)

Reply:
removing it for now .. but it would be nice to name it what you want ...

Reply:
Alice refers to a specific AIML set.  IF we want to give this one a name.. it should simply be ChatGPT...

Reply:
I just removed it for now ... there are some meaningful features  (sleep, wake, bot name, session name, user name)
which would be useful to implement - but this is only the initial implementation